index.html

```diff
@@ -1496,6 +1496,149 @@ <h3>Types</h3>
 
       </section>
 
+      <section>
+        <h3>Names and Descriptions</h3>
+
+        <p>
+When displaying a <a>credential</a>, it can be useful to have
+text provided by the <a>issuer</a> that provides the
+<a>credential</a> with a name as well as a short description of the
+purpose of the <a>credential</a>. The `name` and `description` <a>properties</a>
+are provided for these purposes.
+        </p>
+
+        <dl>
+          <dt><dfn class="lint-ignore">name</dfn></dt>
+          <dd>
+An OPTIONAL property that expresses the name of the <a>credential</a>. If
+present, the value of the <code>name</code> <a>property</a> MUST be a string or
+an object. If the value is an object, the object MUST contain a `value` property
+whose value is a string, and SHOULD contain a `lang` property whose value is a
+`langtag` string defined in [[BCP47]], and MAY contain a `dir` property whose
```

Comment:
Is `langtag` meant to reference the ABNF in RFC5646 (the language tag part of BCP47)? If so, it should be `Language-Tag`, as the `langtag` production leaves out private use and grandfathered productions.

I would probably be more generic here and _also_ ensure that the level of conformance with BCP47 is called out:

```suggestion
string containing a well-formed language tag as defined by [[BCP47]], and MAY
```

Reply:
@aphillips suggestion is good, but the text following it was truncated, so I make this suggestion, fixing that truncation.
```suggestion
string containing a well-formed language tag as defined by [[BCP47]],
and MAY contain a `dir` property whose
```